MAP/src/utils/utils.js

```diff
@@ -0,0 +1 @@
+ export const  toBool = string => typeof(string) == Boolean ?  string : string === 'true' ? true : false;
```

Comment:
SonarCloud doesn't like this as one liner: https://sonarcloud.io/project/issues?id=TheIOFoundation_ProjectLockdown&open=AXmyerDVKA5wYcKzouYR&pullRequest=383&resolved=false&types=CODE_SMELL